Cargo.lock

```diff
@@ -215,22 +215,25 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "7862e21c893d65a1650125d157eaeec691439379a1cee17ee49031b79236ada4"
 dependencies = [
  "proc-macro-error",
- "proc-macro2 1.0.36",
- "quote 1.0.14",
- "syn 1.0.85",
+ "proc-macro2 1.0.37",
+ "quote 1.0.18",
+ "syn 1.0.91",
 ]
 
 [[package]]
 name = "autocfg"
-version = "0.1.7"
+version = "0.1.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "1d49d90015b3c36167a20fe2810c5cd875ad504b39cff3d4eae7977e6b7c1cb2"
+checksum = "0dde43e75fd43e8a1bf86103336bc699aa8d17ad1be60c76c0bdfd4828e19b78"
+dependencies = [
+ "autocfg 1.1.0",
```

Comment:
This is very strange... how come `autocfg 0.1.8` depends on `autocfg 1.1.0`?

Reply:
https://docs.rs/autocfg/0.1.8/autocfg/

> As of autocfg 0.1.8, this is simply re-exporting from autocfg 1.1.0 or later. Please upgrade and refer to the newer version for current documentation!

Reply:
Joyful